bioptim/examples/getting_started/example_soft_contact.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
```

Comment:
Please add a description of what this does (see other examples)
And don't forget to add your new examples to the example GUI

Reply:
Done